module/zfs/zvol.c

```diff
@@ -56,9 +56,11 @@
 
 unsigned int zvol_inhibit_dev = 0;
 unsigned int zvol_major = ZVOL_MAJOR;
+unsigned int zvol_threads = 32;
```

Comment:
It seems to me that reflecting the current value of zvol_threads via blk_update_nr_requests() would make this more consistent with other blk devices.
A stretch goal, which is fine to defer as later work, is to allow zvol_threads to be changed via nr_requests in sysfs.